apisix/control/v1.lua

```diff
@@ -62,27 +64,20 @@ function _M.schema()
 end
 
 
+local healthcheck
 local function extra_checker_info(value, src_type)
```

Comment:
No, it's used to retrieve info by resource type. The format of the results for each resource type from the health check is unified and contains the resource type and resource id. But here, the `src_type` argument is necessary because the resource types are independent and you need to retrieve them separately.

Reply:
The `src_type` is never used in this function